doc/api/dns.md

```diff
@@ -87,11 +88,17 @@ Alternatively, `options` can be an object containing these properties:
 All properties are optional. An example usage of options is shown below.
 
 ```js
-{
-  family: 4,
+const options = {
+  family: 6,
   hints: dns.ADDRCONFIG | dns.V4MAPPED,
-  all: false
-}
+};
+dns.lookup('example.com', options, (err, address, family) =>
+  console.log('address: %j family: IPv%s', address, family));
+// address: "2606:2800:220:1:248:1893:25c8:1946" family: IPv6
+options.all = true;
```

Comment:
Can you add a blank line and a comment here saying something like:

```js

// When options.all = true, the result will be an Array
```

Just to make the example a bit clearer.